crates/typst-syntax/src/kind.rs

```diff
@@ -275,6 +273,11 @@ pub enum SyntaxKind {
     Error,
     /// The end of token stream.
     End,
+
+    /// A path to an imported name from a submodule: `a.b.c`.
+    ImportItemPath,
+    /// A renamed import item: `a as d`.
+    RenamedImportItem,
```

Comment:
After you've merged with main, these can be moved back up to the original spot. I've moved the problematic comment syntax kinds instead, in #4287.